### DISCUSSION TITLE
Next.js app redirects to error page on load  -getInitialProps()

### DISCUSSION THREAD

**Discussion by CodysseusX** [vercel/next.js#15701] [Help]
Next.js v: 9.1.6
React: 16.12.0
Deployment environment: Amazon Cloudfront

The issue started yesterday. On page load (using getInitialProps()), the index page would display error "200. An unexpected error has occurred." When I go to a different page and then back to the index page, it renders the page normally (a server-side vs client side rendering issue perhaps?) 

The same issue was observed again in a different app, where on load it redirects to /_error.

No reason whatsoever for these errors. I have used the same code as a template dozens of times to deploy micro web applications with no issues. Nothing has changed since the last successful deployment. Also, no errors in the console or red flags in the Networks tab of devTools. 

Please note that this happens only when deployed. When running locally there are no issues whatsoever.

**Comment by jamesmosier** [comment]
Do you happen to have a repository with a code sample you could share?

**Comment by CodysseusX** [comment]
I don't, but it's a problem with the node modules dependencies. I deployed using the node modules from a previous project (2 weeks old approximately) and it solved the problem.

**Comment by CodysseusX** [comment]
This might help. Our dependencies:
"dependencies": {
        "@zeit/next-sass": "^1.0.1",
        "aws-amplify": "^2.2.2",
        "axios": "^0.19.0",
        "dotenv": "^8.2.0",
        "next": "^9.1.6",
        "next-page-transitions": "^1.0.0-beta.2",
        "node-sass": "^4.13.0",
        "nprogress": "^0.2.0",
        "react": "^16.12.0",
        "react-dom": "^16.12.0"
    },
    "devDependencies": {
        "@types/react": "^16.9.16",
        "serverless-next.js": "^1.7.0",
        "typescript": "^3.7.3"
    }

**Comment by jeffmarshall** [comment]
I'm having the same issue. Bump!
